crawler.py

```diff
@@ -248,6 +252,14 @@ def start_driver(self):
             opts.add_argument('--no-sandbox')
             opts.add_argument("--load-extension=" + new_extension_path)
 
+            if self.load_extension:
+                try:
+                    opts.add_extension(self.load_extension)
+                except OSError as e:
```

Comment:
If `--load-extension` was provided but it points to a non-existent file, why continue? Is this a recoverable error?

Reply:
I guess if the sole intention of the scan is to have an extension run in parallel to pb, then we'd want to stop it. Otherwise it's able to continue regardless of whether or not the parallel extension loads properly. I figured logging the error was enough to cover this concern, but if you think it's necessary I'm fine adding it in

Reply:
Right, I think if you provide the `--load-extension` option, you want to run a scan with that extension, or not at all.

Reply:
Cool beans, the latest update will break the scan if the `--load-extension` argument doesn't work